requirements.md

```diff
@@ -112,8 +112,8 @@
 - **Trigger:**  User clicks on profile icon
  
 - **Primary Sequence:**
-  1. System identifies their account
-  2. System navigates user to their personal user profile
+  1. User is redirected to their profile page.
+  2. System prompts user with a welcome message.
```

Comment:
Maybe say "System shows a welcome message to user" instead of "..prompts.." because we are not asking the user to give any input.